test/test_system.jl

```diff
@@ -51,15 +51,53 @@
                             "rating")
     @test forecast isa Deterministic
 
-    @test get_forecasts_horizon(sys) == 24
+    horizon = get_forecasts_horizon(sys)
+    @test horizon == 24
     @test get_forecasts_initial_time(sys) == Dates.DateTime("2020-01-01T00:00:00")
     @test get_forecasts_interval(sys) == Dates.Hour(0)
```

Comment:
Seems that get_forecast_interval will only make sense now if we have more than one forecast or the forecasts are in chunks.